docusaurus/docs/dev-docs/deployment/snippets-proxy/admin-redirect.md

```diff
@@ -26,3 +26,39 @@ module.exports = (_config, { strapi }) => {
 	strapi.server.routes(redirects);
 };
 ```
+
+### Shipping your own pages on / instead of the strapi landing page
+
+If you do not wish to have the default landing page mounted on `/` you can create a custom middleware using the sample code below to automatically redirect to your custom `index.html` in the `./public` folder.
+
+```js title="./config/middlewares.js"
+module.exports = ({ env }) => [
+	// ...
+	{ resolve: '../src/middlewares/root' },
```

Comment:
```suggestion
"global::admin-redirect",

Reply:
I get `Error: Middleware global::admin-redirect not found` when I try this

Reply:
You need to change the name of your actual middleware.